doc/installation_jetson.md

```diff
@@ -93,12 +92,10 @@ You can also double-TAB after typing `rs-` to see the full list of SDK examples.
 
 ⮕ Use the V4L Native backend by applying the kernel patching
 
-The method was verified with **Jetson AGX** boards with JetPack **4.4**[L4T 32.4.3], **4.6.1**[L4T 32.7.1] and **5.0.2**[L4T 35.1.0].
+The method was verified with **Jetson Orin™** with JetPack 6.0, **Jetson AGX** boards with JetPack **4.2.3**[L4T 32.2.1,32.2.3], **4.3**[L4T 32.3.1], **4.4**[L4T 32.4.3], **4.6.1**[L4T 32.7.1] and **5.0.2**[L4T 35.1.0].
```

Comment:
Can you please reduce the tested version of the JP?
`**4.4**[L4T 32.4.3], **4.6.1**[L4T 32.7.1] and **5.0.2**[L4T 35.1.0].`
I did the same on the public website

Reply:
Orin is also AGX
So I suggest:
`The method was verified with **Jetson AGX Orin™** with JetPack 6.0, **Jetson Xavier AGX** boards`

Reply:
done